Docker/Dockerfile

```diff
@@ -0,0 +1,76 @@
+FROM ubuntu:22.04
+
+LABEL author="OPERA ADT" \
+      description="RTC interface release" \
+      version="interface_0.1"
+
+RUN apt-get -y update &&\
+    apt-get -y install curl git zip make &&\
+    adduser --disabled-password rtc_user
+
+RUN mkdir -p /home/rtc_user/OPERA/RTC
+COPY . /home/rtc_user/OPERA/RTC
+RUN chown -R rtc_user:rtc_user /home/rtc_user/OPERA
+USER rtc_user 
+
+ENV CONDA_PREFIX=/home/rtc_user/miniconda3
+
+WORKDIR /home/rtc_user
+RUN curl -sSL https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -o miniconda.sh &&\
+    bash miniconda.sh -b -p ${CONDA_PREFIX} &&\
+    rm ${HOME}/miniconda.sh
+
+ENV PATH=${CONDA_PREFIX}/bin:${PATH}
+ENV POSTFIX_ISCE3="rtc"
+ENV ISCEHOME=/home/rtc_user
+
+RUN ${CONDA_PREFIX}/bin/conda init bash
+
+#Install ISCE3 from source code
+RUN echo "Installing Conda environment for ISCE3" &&\
+    mkdir -p ${ISCEHOME}/tools/isce_${POSTFIX_ISCE3}/src &&\
+    mkdir -p ${ISCEHOME}/tools/isce_${POSTFIX_ISCE3}/build &&\
+    mkdir -p ${ISCEHOME}/tools/isce_${POSTFIX_ISCE3}/install &&\
+    cd ${ISCEHOME} &&\
+    curl -sSL https://github.com/gshiroma/isce3/archive/refs/heads/opera_rtc.zip -o isce3_opera_rtc.zip &&\
+    unzip -d ${ISCEHOME}/tools/isce_${POSTFIX_ISCE3} isce3_opera_rtc.zip &&\
+    cd ${ISCEHOME}/tools/isce_${POSTFIX_ISCE3}/src &&\
+    ln -s ../isce3-opera_rtc isce3 &&\
+    rm ${ISCEHOME}/isce3_opera_rtc.zip &&\
+    cd ${HOME} &&\
+    conda env create --file /home/rtc_user/OPERA/RTC/Docker/environment.yml &&\
+    cd ${ISCEHOME}/tools/isce_$POSTFIX_ISCE3/src
+
+SHELL ["conda", "run", "-n", "isce3_rtc", "/bin/bash", "-c"]
+
+ENV CC=${CONDA_PREFIX}/envs/isce3_rtc/bin/x86_64-conda-linux-gnu-gcc
+ENV CXX=${CONDA_PREFIX}/envs/isce3_rtc/bin/x86_64-conda-linux-gnu-g++
+ENV PATH=$PATH:${ISCEHOME}/tools/isce_$POSTFIX_ISCE3/install/bin
+ENV PYTHONPATH=$PYTHONPATH:${ISCEHOME}/tools/isce_$POSTFIX_ISCE3/install/packages
+ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${ISCEHOME}/tools/isce_${POSTFIX_ISCE3}/install/lib
+
+RUN cd ${ISCEHOME}/tools/isce_$POSTFIX_ISCE3/build &&\
+    cmake -DCMAKE_INSTALL_PREFIX=${ISCEHOME}/tools/isce_${POSTFIX_ISCE3}/install -DWITH_CUDA=OFF ${ISCEHOME}/tools/isce_${POSTFIX_ISCE3}/src/isce3 &&\
+    make -j16 VERBOSE=ON &&\
+    make install &&\
+    cd /home/rtc_user/miniconda3/envs/isce3_rtc/lib &&\
+    ln -s ${ISCEHOME}/tools/isce_${POSTFIX_ISCE3}/install/lib/libisce3.so &&\
+    echo 'ISCE3 installation successful!'
+
+RUN echo "Installing OPERA s1-reader and RTC" &&\
+    mkdir -p $HOME/OPERA &&\
+    cd $HOME/OPERA &&\
+    curl -sSL https://github.com/seongsujeong/s1-reader/archive/refs/heads/correction_and_calibration.zip -o s1_reader.zip &&\
```

Comment:
@seongsujeong Same as my comment above for ISCE3, we should use a released version. Is it because your PRs are not merged and you want to use the latest and greatest? Remember that this is for Interface delivery and all features are not needed. We don't really need noise correction or any advanced feature here.

Reply:
Yes, we need it for thermal noise correction and also abs. radiometric calibration.  @seongsujeong , are there other differences from your `corrrection_and_calibration` branch and the S-1 reader main branch?

@hfattahi , do you prefer that we turn off both corrections and use the S-1 reader main branch?

Reply:
@gshiroma @hfattahi I have added few changes to populate some metadata, which are nose and calibration annotation files. I've tested the current RTC workflow with the main branch of `opera-adt/s1-reader`, but it was not successful because of that.